src/persist/src/lib.rs

```diff
@@ -56,6 +55,14 @@ pub mod storage;
 // - Failure while draining from buffer into future.
 // - Equality edge cases around all the various timestamp/frontier checks.
 
+/// A type usable as a persisted key or value.
+///
+/// TODO: Replace Abomonation with something like an Encode+Decode trait.
+///
+/// TODO: Can we get rid of Debug + Ord? They're only here for debug assertions.
```

Comment:
I think this is totally fine because differential also imposes Debug + Ord constraints on key / val types - see: https://docs.rs/differential-dataflow/0.12.0/src/differential_dataflow/lib.rs.html#87

Ord seems inevitable at least because things have to be sorted before being placed in a trace unless I'm missing something?